### PR TITLE
Support generate demos bundles for publish

### PR DESCRIPTION
**Pull request by OlegKipchatov**

### CODE REVIEW

File: utils/create-bundles/React/bundler.ts [hunk 1]
```diff
@@ -0,0 +1,52 @@
...
+      outdir: destinationDemoPath,
+      entryPoints: this.#getEntryPoints(sourceDemoPath),
+      plugins: [ignoreMissingCss],
+      external: ['*.png', '*.svg'],
+    };
+
+    return options;
+  };
+
+  buildDemo = async (demo: Demo): Promise<void> => {
```
Comment (VasilyStrelyaev) [inline utils/create-bundles/React/bundler.ts]: this method looks identical for both classes - can we avoid duplicating it?

Comment (OlegKipchatov) [reply utils/create-bundles/React/bundler.ts]: I'm afraid that in the future we will need to make changes to each method separately

Comment (VasilyStrelyaev) [reply utils/create-bundles/React/bundler.ts]: I'm not sure this is a good reason. The general rule of thumb is to write the most efficient and simple code at present, and not make it redundant/non-optimal at present for the sake of probable new circumstances in the future.

**Review by VasilyStrelyaev — COMMENTED** [review]
👆

File: utils/create-bundles/React/bundler.ts (hunk 1 above)
Comment (OlegKipchatov) [reply utils/create-bundles/React/bundler.ts]: Okay, I prepare commit with this https://github.com/DevExpress/devextreme-demos/pull/2851/commits/dac512795cea7c2c8a1fb65e438bc062ad58b81f

**Review by VasilyStrelyaev — DISMISSED** [review]

**Review by VasilyStrelyaev — DISMISSED** [review]

**Pull request merged**